dmotion:
  com_z:
    data: [0, 1, 1.96, 2.9900000000000002, 4, 5.5, 28, 28.150000000000002, 28.280000000000008, 28.250000000000004, 28.100000000000001, 28, 0.0001, 0.2772168561560146, 0.043761133973749262, -0.043761133973749262, -0.12268305454440676, -0.0001]